qutebrowser/config/configdata.yml

```diff
@@ -1811,6 +1811,21 @@ input.match_counts:
     Disabling it allows for emacs-like bindings where number keys are passed
     through (according to `input.forward_unbound_keys`) instead.
 
+input.mode_override:
+  default: null
+  type:
+    name: String
+    none_ok: true
+    valid_values:
+      - normal
+      - insert
+      - passthrough
+      - caret
```

Comment:
Given that @toofar said this doesn't work consistently (I haven't tried myself), and I'm not sure how useful it really is, I wonder if we should support this? Easier to add it in the future if people ask for it than to take it away.

Reply:
Fair enough, removed